src/CalendarDate.ts

```diff
@@ -424,4 +424,18 @@ export class CalendarDate {
       )
     );
   }
+
+  /**
+   * Determines the quarter of the year (1-4) based on the month of the date.
```

Comment:
```suggestion
   * The quarter of the year (1-4) based on the month of the date.
```
since this is a getter, in the usage this seem like a variable and not a function